pyFAST/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -1776,43 +1936,46 @@ def _FF_setup_TS(self):
         
                     # Open TurbSim outputs for the Low box and one High box (they are all of the same size)
                     lowbts = TurbSimFile(os.path.join(seedPath,'TurbSim', 'Low.bts'))
-                    highbts  = TurbSimFile(os.path.join(seedPath,'TurbSim', f'HighT1.bts'))
+                    highbts  = TurbSimFile(os.path.join(seedPath,'TurbSim', 'HighT1.bts'))
         
                     # Get dictionary with all the D{X,Y,Z,t}, L{X,Y,Z,t}, N{X,Y,Z,t}, {X,Y,Z}0
                     dt_low_desired = self.Cmeander*D_/(10*Vhub_) # will be made multiple of dT_High inside _getBoxesParamsForFF
                     #d = self._getBoxesParamsForFF(lowbts, highbts, dt_low_desired, D_, HubHt_, xWT, yt)
-                    d = self._getBoxesParamsForFF(lowbts, highbts, self.dt_low_les, D_, HubHt_, xWT, yt)
+                    d = self._getTSBoxesParamsForFF(lowbts, highbts, self.dt_low_les, D_, HubHt_, xWT, yt)
         
                     # Write the file
+                    xWT=xWT+d['dX_Low']
                     writeFastFarm(outputFSTF, templateFSTF, xWT, yt, zWT, d, OutListT1=self.outlistFF, noLeadingZero=True)
         
         
                     # Open saved file and change additional values manually or make sure we have the correct ones
                     ff_file = FASTInputFile(outputFSTF)
-                    ff_file['InflowFile'] = f'"./{self.IWfilename}"'
+                    ff_file['InflowFile'] = '"InflowWind.dat"'
                     #ff_file['DT']=1.0
                     ff_file['Mod_AmbWind'] = 3  # 1: LES boxes; 2: single TurbSim; 3: multiple TurbSim
-                    ff_file['TMax'] = self.tmax
+                    ff_file['TMax'] = self.tmax* self.tsim
         
                     # Super controller
                     ff_file['UseSC'] = False
                     ff_file['SC_FileName'] = '/path/to/SC_DLL.dll'
         
                     # Wake dynamics
                     ff_file['Mod_Wake'] = self.mod_wake
-                    if self.mod_wake == 1: # Polar model
-                        self.dr = self.cmax
-                    else: # Curled; Cartesian
-                        self.dr = round(self.D/10)
+                    # if self.mod_wake == 1: # Polar model
+                    #     self.dr = self.cmax
+                    # else: # Curled; Cartesian
+                    #     self.dr = round(self.D/10)
                     ff_file['dr'] = self.dr
                     ff_file['NumRadii']  = int(np.ceil(3*D_/(2*self.dr) + 1))
                     #ff_file['NumPlanes'] = int(np.ceil( 20*D_/(dt_low_desired*Vhub_*(1-1/6)) ) )
-                    ff_file['NumPlanes'] = int(np.ceil( 20*D_/(self.dt_low_les*Vhub_*(1-1/6)) ) )
+                    # ff_file['NumPlanes'] = int(np.ceil( self.extent_low[1]*D_/(self.dt_low_les*Vhub_*(1-1/6)) ) )
+                    ff_file['NumPlanes'] = int(np.ceil( self.extent_low[1]*D_/(self.dt_low_les*Vhub_ ) ))
```

Comment:
This is not what is on the modeling guidance.

Reply:
No it is the same. This just fits the low and high resolution grids. This assures that the origin of the high resolution grid is a node in the low resolution grid.